packages/web-components/src/image/image.styles.ts

```diff
@@ -26,14 +32,16 @@ export const styles = css`
   :host([fit='none']) ::slotted(img) {
     object-fit: none;
     object-position: top left;
-    height: 100%;
-    width: 100%;
+  }
+  :host([fit='center']) {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    height: 200px;
```

Comment:
The height is *fixed* via styles for any image with this set? This seems incorrect…

Reply:
Oops, forgot to remove that. Yes, it's not "*fixed* fixed" in this case.